app/assets/javascripts/discourse/app/components/user-menu/reviewable-queued-post-item.js

```diff
@@ -0,0 +1,22 @@
+import UserMenuDefaultReviewableItem from "discourse/components/user-menu/default-reviewable-item";
+import I18n from "I18n";
+import { htmlSafe } from "@ember/template";
+
+export default class UserMenuReviewableQueuedPostItem extends UserMenuDefaultReviewableItem {
+  get actor() {
+    return I18n.t("user_menu.reviewable.queue");
+  }
+
+  get description() {
+    const title = this.reviewable.topic_title;
+    if (this.reviewable.is_new_topic) {
+      return htmlSafe(title);
+    } else {
+      return I18n.t("user_menu.reviewable.new_post_in_topic", { title });
```

Comment:
Just want to confirm that the emojis in title will be displayed here?

Reply:
I discovered it was possible to trigger an XSS here while testing to see if emojis would show up, so I've fixed the XSS and made sure emojis will show up.